hlxsites/merative2
# fix(state-campaign): address design QA issues seen on the Curam's Missouri state page

## Issue

Fixes - https://jira.sdlc.merative.com/browse/MERATIVE-880

## Description

**Changed**

- This PR fixes issues that came out of the design QA of the Curam's Missouri state page/campaign
- **Note:** The video lightbox/model feature will be addressed in a different ticket (https://jira.sdlc.merative.com/browse/MERATIVE-858). In the meantime, any of the CTAs that are linking out to YT, etc will be opened in a new tab.

## Design Specs

- Figma Link - https://www.figma.com/file/lcNLadWizxvzfhBpqds4hU/Curam---State-Microsite?type=design&node-id=664%3A1613&mode=design&t=BbUoqrz9MjDWQH1J-1

## Test URLs

![Screenshot 2023-09-08 at 1 13 33 PM](https://github.com/hlxsites/merative2/assets/1815714/7a68a12a-466a-4d6e-a9dc-0cdb4e0e42c5)
  
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/campaigns/curam/missouri
- After (Changes from this PR): https://fix-design-qa-880--merative2--proeung.hlx.page/campaigns/curam/missouri
  
## Testing Instruction

- Visit this deploy preview page (https://fix-design-qa-880--merative2--proeung.hlx.page/campaigns/curam/missouri)
- Ensure that each band/pattern is displaying correctly for all breakpoints.
